modules/openshift-developer-cli-installing-odo-on-windows.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * assebmly_installing-odo.adoc     
+
+[id="installing-odo-on-windows"]
+
+= Windows
+
+. Download the latest file from the Bintray link:https://dl.bintray.com/odo/odo/latest/windows-amd64/odo.exe[odo.exe]
+or from the latest release page on link:https://github.com/openshift/odo/releases[GitHub].
+. Extract the file and add the location of extracted binary to your `GOPATH/bin` directory (see below to create a Go binary directory)
+
+== Setting the `PATH` variable for Windows 7/8
+
+The following example demonstrates how to set up a path variable. Your binaries can be located in any location but for the purpose of this example we will use `C:\go-bin` as the location.
+
+. Create a folder at `C:\go-bin`.
+. Right click on *Start* and click on *Control Panel*.
```

Comment:
Through all these steps, remove "on" after "click". Should just be `Right click *Start* and click *Control Panel*.` according to IBM style guide.